ckanclient/upload.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import requests
+
+
+class CkanUploadAPIError(Exception):
+    pass
+
+
+def push_data_to_blob_storage(action, resource):
+    """Send a POST request with specific payload to the URL given by the Batch
+    API response."""
+    response = requests.put(
+        action["href"],
```

Comment:
personally i would make these arguments explicit in the overall method signature rather than embedding them in `action` dict (but i may have missed something)